src/pathfinding_service/model/token_network.py

```diff
@@ -117,6 +117,27 @@ def handle_channel_closed_event(self, channel_identifier: ChannelIdentifier):
                 channel_identifier=channel_identifier,
             )
 
+    def get_channel_view_perspective(
```

Comment:
what's the idea behind *perspective*?

Reply:
I thought the channel view with respect to the perspective of the updating participant, makes sense?

Reply:
What about ` get_channel_views_for_partner`?

Reply:
good suggestion changed it